js/form.js

```diff
@@ -268,7 +268,16 @@
 
     $('#petition_info .petition-details').replaceWith(petition_details);
     $('#petition_info').show();
-  }
+
+    // returns the status of petition i.e. 'closed', 'open'
+    var petitionStatus = PetitionMap.current_petition.data.attributes.state
+
+    if (petitionStatus !== 'open') {
+      $( "#sign_petition_btn" ).hide();
+    } else {
+      $( "#sign_petition_btn" ).show();
+      }
+    }
```

Comment:
I'd flip the conditional around like this:

``` javascript
if (petitionStatus == 'open') {
  $('#sign_petition_btn').show();
} else {
  $('#sign_petition_btn').hide();
}
```

Also seems like there's an extra `}` in there?